modules/tinymce/src/plugins/quickbars/main/ts/api/Commands.ts

```diff
@@ -0,0 +1,25 @@
+import Editor from 'tinymce/core/api/Editor';
+
+import * as Actions from '../insert/Actions';
+import * as Conversions from '../insert/Conversions';
+import * as Picker from '../insert/Picker';
+
+const register = (editor: Editor): void => {
+  if (!editor.queryCommandSupported('PickFile')) {
```

Comment:
I think we need to make the command quickbar specific so maybe `QuickbarInsertImage`? It does more than just pick a file it also inserts the image.

- [ ] Make the command more specific to the quickbar plugin

Reply:
updated